os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -238,6 +238,34 @@ static inline void mpu_showtype(void)
 #endif
 }
 
+/****************************************************************************
+ * Name: mpu_get_access_type
+ *
+ * Description:
+ *   Get the mpu region access type
+ *
+ ****************************************************************************/
+
+static inline char *mpu_get_access_type(uint32_t rasr)
+{
+	uint32_t val = rasr & MPU_RASR_AP_MASK;
+	if (val == MPU_RASR_AP_NONO) {
+		return "NO/NO";
+	} else if (val == MPU_RASR_AP_RONO) {
+		return "RO/NO";
+	} else if (val == MPU_RASR_AP_RWNO) {
+		return "RW/NO";
+	} else if (val == MPU_RASR_AP_RORO) {
+		return "RO/RO";
+	} else if (val == MPU_RASR_AP_RWRO) {
+		return "RW/RO";
+	} else if (val == MPU_RASR_AP_RWRW) {
+		return "RW/RW";
+	} else {
+		return "XX/XX";
+	}
```

Comment:
Switch statement provides better performance if the number of switches are more (more than 5). But in this case, if-else also provides similar performance. And since, it is just a return statement, I think this method provides simpler code and readablility.